src-built-in/components/badBoy/index.html

```diff
@@ -1,35 +1,35 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset="utf-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>Stress Test</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="https://blueprintjs.com/docs/docs-app.css" rel="stylesheet" />
-  <link rel="stylesheet" type="text/css" media="screen" href="https://mere.github.io/stress-test/main.css" />
-  <script src="https://mere.github.io/stress-test/main.js"></script>
-</head>
-<body class="bp3-dark bp3-card bp3-elevation-0 bp3-running-text main">
-  <div class="bp3-spinner"><div class=""><svg width="50" height="50" stroke-width="8.00" viewBox="1.00 1.00 98.00 98.00"><path class="bp3-spinner-track"  d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"></path><path class="bp3-spinner-head" id="spinner" d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90" pathLength="280" stroke-dasharray="280 280" stroke-dashoffset="210"></path></svg></div></div>
-  <br />
-
-  <figcaption>Single run:</figcaption>
-  <div class="bp3-button-group .modifier">
-    <a class="bp3-button bp3-icon-time" id="single2" role="button">2 seconds</a>
-    <a class="bp3-button bp3-icon-time" id="single5" role="button">5 seconds</a>
-    <a class="bp3-button bp3-icon-time" id="single10" role="button">10 Seconds</a>
-  </div>
-
-  <br />
-  <br />
-  <figcaption>Continuous: (2 second break intervals)</figcaption>
-  <div class="bp3-button-group .modifier">
-    <a class="bp3-button bp3-icon-repeat" id="multi2" role="button">2 seconds</a>
-    <a class="bp3-button bp3-icon-repeat" id="multi5" role="button">5 seconds</a>
-    <a class="bp3-button bp3-icon-repeat" id="multi10" role="button">10 Seconds</a>
-  </div>
-  <br />
-
-  <pre id="message"></pre>
-</body>
+<!DOCTYPE html>
```

Comment:
This file got into the PR somehow. Can you get it out again? I tried to `git checkout master src-built-in/components/badBoy` to fix it, but git said the file didn't exist.